man/systemd.unit.xml

```diff
@@ -188,8 +188,8 @@
     suffix is <filename>.requires/</filename> in this case.</para>
 
     <para>Along with a unit file <filename>foo.service</filename>, a "drop-in" directory
-    <filename>foo.service.d/</filename> may exist. All files with the suffix
-    <literal>.conf</literal> from this directory will be parsed after the file itself is
+    <filename>foo.service.d/</filename> may exist. All files with the suffix <literal>.conf</literal>
+    from this directory will be parsed in order of their filename after the file itself is
```

Comment:
I think the discussion of the order of parsing should go at the end of this paragraph:
> In addition to <filename>/etc/systemd/system</filename>, the drop-in <literal>.d</literal>
    directories for system services can be placed in <filename>/usr/lib/systemd/system</filename> or
    <filename>/run/systemd/system</filename> directories. Drop-in files in <filename>/etc</filename>
    take precedence over those in <filename>/run</filename> which in turn take precedence over those
    in <filename>/usr/lib</filename>. Drop-in files under any of these directories take precedence
    over unit files wherever located.